client/stylesheets/_toCat.scss

```diff
@@ -5599,4 +5600,128 @@ textarea {
     border-radius: 4px !important;
 }
 
+/*~~~~~~~~~ Default Profile Message ~~~~~~~~~*/
+
+.defaultProfileDiv {
+    text-align: center;
+    margin-top: 120px !important;
+}
+
+.defaultProfileDiv h2 {
+    font-weight: 400 !important;
+    color: #555 !important;
+    vertical-align: middle !important;
+}
+
+.defaultProfileDiv h2 i {
+    margin-right: 15px;
+    vertical-align: middle !important;
+    padding: 10px !important;
+    background-color: #f8504b !important;
+    color: #fff !important;
+
+}
+
+.defaultProfileDiv p {
+    color: #666 !important;
+}
+
+.defaultProfileButton {
+    margin-top: 30px;
+    list-style: none !important;
+}
+
+.defaultProfileButton a {
+    padding: 12px 12px;
+    font-size: 0.9rem;
+    color: #fff;
+    background-color: #f8504b;
+    border-radius: 4px;
+}
+
+.defaultProfileButton a:hover {
+    background-color: #f73c37 !important;
+    color: #fff !important;
+}
+
+/*~~~~ Home No Listings ~~~~*/
+
+.homeNoListings {
+    text-align: center;
+    padding-top: 10px;
+} 
+
+.homeNoListings h2 {
+    font-weight: 400 !important;
+    color: #555 !important;
+    font-size: 1.6rem !important;
+}
+
+.homeNoListings p {
+    width: 70%;
+    margin: 10px auto !important;
+    color: #666 !important;
+}
+
+.homeNoListingsButton {
+    margin-top: 40px !important;
+    list-style: none !important;
+}
+
+.homeNoListingsButton a {
```

Comment:
Merge rule `.homeNoListingsButton a` with rule on line 5666<br>Selector `homeNoListingsButton` should be written in lowercase with hyphens